README.md

```diff
@@ -233,7 +233,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [H2](http://h2database.com/) - Small SQL database notable for its in-memory functionality.
 * [HikariCP](https://github.com/brettwooldridge/HikariCP) - High-performance JDBC connection pool.
 * [JDBI](http://jdbi.org/) - Convenient abstraction of JDBC.
-* [Jedis](https://github.com/xetorthio/jedis) - A small client for interaction with Redis, with methods for commands.
+* [Jedis](https://github.com/xetorthio/jedis) - A small client for interaction with redis, with methods for commands.
+* [Jest](https://github.com/searchbox-io/Jest) - Elasticsearch REST Client, supports JSON queries and also has own Java DSL.
```

Comment:
Hi @heldev, would you kindly revert "redis" back to "Redis" please? AFAICT, "Redis" seems to be used more in the Redis documentation than "redis", even though the term "redis" is used in the website title header.

I've also spotted a couple of other minor spelling/grammatical errors that could be fixed, e.g.: "Elasticsearch" -> "ElasticSearch", and "Client" to "client".